source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -140,11 +140,17 @@ Consistency
 ~~~~~~~~~~~
 
 ``mongosync`` supports :term:`eventual consistency` on the destination
-cluster. Read Consistency is not guaranteed on the destination cluster,
-the source and destination clusters may differ at a given point in
+cluster. Read consistency is not guaranteed on the destination cluster.
+The source and destination clusters may differ at a given point in
 time. However, if new writes are halted on the source cluster, the
 source and destination clusters will eventually hold the same data.
 
+.. warning:: 
```

Comment:
BTW here's the blurb we had been discussing to share with a customer who wanted to use mongosync for continuous sync:

> mongosync must [commit](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/api/commit/) for the destination cluster to have an up to date, consistent view of the source cluster’s data. If a disaster occurs and shuts down the source cluster before mongosync can commit, the destination cluster will not have a consistent snapshot of the source data. For any continuous synchronization use cases with mongosync, ensure mongosync commits before cutting over from the source to the destination.
When used with sharded clusters, balancers need to be paused on both the source and the destination for the lifetime of the sync. If mongosync is used for continuous synchronization, this means that the balancers will remain disabled on either cluster.

Can we have a 'considerations for continuous sync' section?

Reply:
I'll slot this in! Thanks for sending the copy over.

Reply:
Thanks! If we could also add ", and there may be torn transactions." to the "[...] will not have a consistent snapshot of the source data" sentence that would be great